models/cycle_gan_semantic_model.py

```diff
@@ -66,22 +66,22 @@ def __init__(self, opt):
         # Code (paper): G_A (G), G_B (F), D_A (D_Y), D_B (D_X)
         self.netG_A = networks.define_G(opt.input_nc, opt.output_nc,
                                         opt.ngf, opt.netG, opt.norm, 
-                                        not opt.no_dropout, opt.init_type, opt.init_gain, self.gpu_ids)
+                                        not opt.no_dropout, opt.G_spectral, opt.init_type, opt.init_gain, self.gpu_ids)
```

Comment:
Be careful that these changes may not belong to this PR.

Reply:
True, they should have been in a fix PR.